physics/GFS_suite_interstitial.F90

```diff
@@ -742,16 +742,16 @@ subroutine GFS_suite_interstitial_4_run (im, levs, ltaerosol, cplchm, tracers_to
           if (imp_physics == imp_physics_thompson .and. (ntlnc>0 .or. ntinc>0)) then
             do k=1,levs
               do i=1,im
-                !> - Density of air in kg m-3
-                rho_dryair(i,k) = prsl(i,k) / (con_rd*save_tcp(i,k))
                 !> - Convert specific humidity to dry mixing ratio
                 qv_mp(i,k) = spechum(i,k) / (one-spechum(i,k))
+                !> - Density of air in kg m-3
+                rho_air(i,k) = 0.622*prsl(i,k) / (con_rd*save_tcp(i,k)*(qv_mp(i,k)+0.622))
```

Comment:
Same comment about memory footprint and possibly using a local scalar variable not arrays.

Reply:
Yes, in this place certainly possible. One could also define another scalar orho = 1/rho and do only one division instead of two (754, 763).